src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -169,6 +176,75 @@ protected void execute(CommandEvent event)
             }
         }
     }
+
+    public class AppendlistqCmd extends OwnerCommand 
+    {
+        public AppendlistqCmd()
+        {
+            this.name = "appendq";
+            this.aliases = new String[]{"addq"};
+            this.help = "appends songs in queue to an existing playlist";
+            this.arguments = "<name>";
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(CommandEvent event) 
+        {
+            String[] parts = event.getArgs().split("\\s+", 2);
+            if(parts.length<1)
+            {
+                event.reply(event.getClient().getError()+" Please include a playlist name to add!");
+                return;
+            }
+            String pname = parts[0];
+            Playlist playlist = bot.getPlaylistLoader().getPlaylist(pname);
+            if(playlist==null)
+                event.reply(event.getClient().getError()+" Playlist `"+pname+"` doesn't exist!");
+            else
+            {
+                AudioHandler ah = (AudioHandler)event.getGuild().getAudioManager().getSendingHandler();
+                List<QueuedTrack> list = ah.getQueue().getList();
+                if(list.isEmpty())
+                {
+                    Message nowp = ah.getNowPlaying(event.getJDA());
+                    if(nowp==null)
+                    {
+                        event.reply(event.getClient().getWarning() + " There is no music in the queue!");
+                        return;
+                    }
+                }
+                List<AudioTrack> al = new ArrayList<>();
+                al.add(ah.getPlayer().getPlayingTrack());
+                for(QueuedTrack q: list)
+                {
+                    al.add(q.getTrack());
+                }
+                StringBuilder builder = new StringBuilder();
+                playlist.getItems().forEach(item -> builder.append("\r\n").append(item));
+                for(AudioTrack url: al)
+                {
+                    if(url==null)
+                    {
+                        continue;
+                    }
+                    String u = url.getInfo().uri;
+                    if(u.startsWith("<") && u.endsWith(">"))
+                        u = u.substring(1, u.length()-1);
```

Comment:
This is not needed. The angle brackets are already removed by the play command:

https://github.com/jagrosh/MusicBot/blob/d5bb16d1436c9e5dcd81df8221d38cb7fa4317c7/src/main/java/com/jagrosh/jmusicbot/commands/music/PlayCmd.java#L87-L89

```suggestion

```